libs/editor/egui_editor/src/input/canonical.rs

```diff
@@ -93,10 +93,6 @@ pub enum Modification {
     ToggleDebug,
     ToggleCheckbox(usize),
     OpenUrl(String),
-    Heading(u32),
-    BulletListItem,
-    NumberListItem,
-    TodoListItem,
```

Comment:
no more one-off modification variants for block styles; these are handled by the generic apply_style fn now